plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -108,6 +154,18 @@ TimingTriggerCandidateMaker::receive_hsievent(ipm::Receiver::Response message)
   auto data = serialization::deserialize<dfmessages::HSIEvent>(message.data);
   ++m_tsd_received_count;
   
+  if (m_hsi_passthrough == true){
```

Comment:
If you just want to check that no bits beyond the low byte are set, you could do it more straightforwardly with, eg: `if (data.signal_map & 0xffffff00 != 0) { // high bits are set }`. I think that would save you the work making `bitset`s

Reply:
Added this, which means I removed most of the nice functions I had fun creating, which are now obsolete :)